README.md.mustache

```diff
@@ -54,7 +54,7 @@
 
 - Author(s):
 {{# authors }}
-  - {{& name }}{{# initial }} (initial){{/ initial }}
+  - {{& name }}{{# email }} <{{& email }}>{{/ email }}{{# initial }} (initial){{/ initial }}
```

Comment:
It was a conscious choice not to include emails in the README, since we don't really want to encourage emailing authors directly when someone is visiting GitHub (they should use issues instead). It was added to the OPAM template since some people had OPAM files where they already listed their emails, and we didn't want information to disappear. I'm personally against adding the email to the README.

Reply:
Makes sense, I reverted this commit